qiskit/providers/models/backendconfiguration.py

```diff
@@ -300,6 +301,10 @@ def __init__(
                 - segment: Segment this processor belongs to within a larger chip.
             parametric_pulses (list): A list of pulse shapes which are supported on the backend.
                 For example: ``['gaussian', 'constant']``
+            measure_esp_enabled (bool): Whether ESP readout is enabled on this device. ESP readout
```

Comment:
```suggestion
            measure_esp_enabled (bool): Whether excited state promoted (ESP) readout is enabled on this device. ESP readout
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6543/commits/de6d6073315d5607ef59c929861b38aa6d47847a.